pkgs/by-name/cv/cvtek/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+let
+  version = "0-unstable-2024-06-08";
+in
```

Comment:
Why the let?

Reply:
To avoid `rec` when the rev is a version tag.